src/logger.js

```diff
@@ -1,15 +1,9 @@
 const pino = require('pino');
 
-const usingElastic = process.env.ELASTIC_APM_SERVER_URL && process.env.ELASTIC_APM_SERVICE_NAME;
-
-// Pick our logger based on whether we're using APM monitoring with ELK
-let logger = usingElastic
-  ? // Structured JSON logging
-    pino(require('@elastic/ecs-pino-format')({ convertReqRes: true }))
-  : // Pretty debug logging
-    pino({
-      prettyPrint: {},
-      prettifier: require('pino-colada'),
-    });
+// Pretty debug logging
+let logger = pino({
+  prettyPrint: {},
+  prettifier: require('pino-colada'),
```

Comment:
In a follow-up, I'd like to fix how we do this.  `pino-collada` is great for debugging, but it's not what we should use in production.  Maybe you can work on that next?

Reply:
Yes, will do, since I'm also working with pino in [#2499](https://github.com/Seneca-CDOT/telescope/issues/2499). I will fix there